crates/red_knot_python_semantic/src/types.rs

```diff
@@ -473,6 +479,7 @@ impl<'db> Type<'db> {
             Type::Unknown => Type::Unknown,
             // TODO intersections
             Type::Intersection(_) => Type::Unknown,
+            Type::Tuple(_) => typeshed_symbol_ty(db, "tuple"),
```

Comment:
This isn't correct: `typeshed_symbol_ty` looks things up in the (fictional) `_typeshed` module in typeshed, but we want to look things up in the `builtins` module:

https://github.com/astral-sh/ruff/blob/62c7d8f6ba118c85e65f79be2f3c0f992ad32b09/crates/red_knot_python_semantic/src/stdlib.rs#L57-L63

```suggestion
            Type::Tuple(_) => builtins_symbol_ty(db, "tuple"),
```